datalad_registry/templates/overview.html

```diff
@@ -178,35 +191,99 @@ <h2>Search query syntax</h2>
     <div class="pager">
       {{ render_pagination_widget(pagination, '.overview') }}
     </div>
+
+
+    <div>
+      <span style="margin-right: 10px">Combined Nr of Annexed
+      files: {{ ((stats.datalad_ds_stats.stats.annexed_file_count if stats.datalad_ds_stats.stats.annexed_file_count else 0) +
+        (stats.pure_annex_ds_stats.annexed_file_count if stats.pure_annex_ds_stats.annexed_file_count else 0))|intcomma }}</span>
+      <span style="margin-right: 5px">Combined Annexed file
+        size: {{ ((stats.datalad_ds_stats.stats.annexed_files_size if stats.datalad_ds_stats.stats.annexed_files_size else 0) + (stats.pure_annex_ds_stats.annexed_files_size if stats.pure_annex_ds_stats.annexed_files_size else 0))|filesizeformat }}</span>
+
+      <!-- Stats Trigger Button -->
+      <button id="ShowStats">Show Details</button>
+    </div>
+
+
+    <!-- Statistics modal -->
+    <div id="StatsModal" class="modal">
+      <div class="modal-content">
+        <div class="modal-header">
+          <span id="CloseStats" class="close">&times;</span>
+          <h2>Stats</h2>
+        </div>
+        <div id="stats" class="modal-body">
+          <ul>
+            <li>
+              <h3>Datalad Datasets Stats</h3>
+              <ul>
+                <li>
+                  <h4>Unique Datalad Dataset Stats</h4>
+                  {{ render_annex_ds_collection_stats(stats.datalad_ds_stats.unique_ds_stats) }}
+                </li>
+                <li>
+                  <h4>Stats without deduplication</h4>
+                  {{ render_annex_ds_collection_stats(stats.datalad_ds_stats.stats) }}
+                </li>
+              </ul>
+            </li>
+            <li>
+              <h3>Pure Annex Dataset Stats</h3>
+              {{ render_annex_ds_collection_stats(stats.pure_annex_ds_stats) }}
+            </li>
+            <li>
+              <h3>Non-Annex Dataset Stats</h3>
```

Comment:
```suggestion
              <h3>Non-annex repositories stats</h3>
```

are those datalad datasets or just repositories or either?

Reply:
They are git repos that don't have a `"git-annex"` branch. Referring them as repos are more precise, but in our code base, we loosely just call all repos datasets. The current arrangement in our code base is a bit confusing.